sync/syncer.go

```diff
@@ -83,6 +83,11 @@ func (s *Syncer) IsSynced() bool {
 	return s.ValidatedLayer()+1 == s.maxSyncLayer()
 }
 
+func (s *Syncer) IsLatest() bool {
```

Comment:
better to change is synced logic it is used in some places
this is basically the same method except the >=

Reply:
notice that this method uses the LatestLayer and not VerifiedLayer, I'm not sure this is the functionality expected from `IsSynced` currently

Reply:
Np, we need one way to do it you can change it if it works

Reply:
Np, we need one way to do it you can change it if it works

Reply:
What are the diffrences between `ValidatedLayer` and `LatestLayer`?

Reply:
`Validated` layer is the last layer validated by the tortoise, `Latest` is the last layer seen on gossip/ sync

Reply:
yuval made me look at this again and i think that if you want to shut down the listener until synced just use isSynced the condition there is even harder

Reply:
I've changed the synched parameter after talking to Yuval, I think ths=is fits here best, @zalmen  can you please clarify?

Reply:
IsSynced == IsLatest, see my fixes